### ISSUE TITLE
copy to clipboard not working since 1.19 on Fedora 40

### ISSUE THREAD

**Issue by rhasti** [pwsafe/pwsafe#1307]
Using Fedora 40. Installed the fedora39 rpm. 
Since version 1.19 the copy to clipboard stopped working. 
Double click to copy password into clipboard is not working.
Autofill feature seems to be gone as well.

```
🐱 rpm -q passwordsafe
passwordsafe-1.19-1.x86_64

🐱 hostnamectl 
     Static hostname: kimba
           Icon name: computer-laptop
             Chassis: laptop 💻
          Machine ID: 750ce5ace5764f1cbc7e5c9802038085
             Boot ID: da8f6b0c8f5b4c099bd2a947ba3740c2
    Operating System: Fedora Linux 40 (Workstation Edition)
         CPE OS Name: cpe:/o:fedoraproject:fedora:40
      OS Support End: Tue 2025-05-13
OS Support Remaining: 10month 4w 1d
              Kernel: Linux 6.8.11-300.fc40.x86_64
        Architecture: x86-64
     Hardware Vendor: Lenovo
      Hardware Model: Legion 7 16ACHg6
    Firmware Version: GKCN64WW
       Firmware Date: Thu 2023-12-07
        Firmware Age: 6month 6d
```

Desktop: Gnome (Wayland)

Tried disabling selinux. Seems that is not the issue.

**Comment by rhasti** [comment]
Gnome with Xorg is not affected.

**Comment by hxu1** [comment]
@rhasti Can you please check if the build from #1310 is working for you?

**Comment by igor2x** [comment]
> Double click to copy password into clipboard is not working.
This is probably because of small bug in code, that @hxu1 now has fixed in its own repository and is waiting for code review and merge into Password Safe repository in PR [1310](https://github.com/pwsafe/pwsafe/pull/1310).

> Autofill feature seems to be gone as well.
Autofill was never working in Password Safe on Wayland. Maybe you have used X.org session before, when this is working fine. This issue was already reported in [issue 1298](https://github.com/pwsafe/pwsafe/issues/1298).

**Comment by rhasti** [comment]
> @rhasti Can you please check if the build from #1310 is working for you?

I confirm copy to clipboard works with these changes applied.

**Comment by igor2x** [comment]
@rhasti, [PR 1310](https://github.com/pwsafe/pwsafe/pull/1310) was merged and new Password Safe 1.19.1 was released today. Can you please [download](https://github.com/pwsafe/pwsafe/releases/tag/1.19.1) latest version and install. I have tested it on Ubuntu 24.04 and I see this issue is fixed.

Can you please provide quick update and if issue is fixed, just close down this issue on Github.

**Comment by rhasti** [comment]
Issue is fixed with new version. Thank you
